includes/html/functions.inc.php

```diff
@@ -175,7 +176,7 @@ function generate_device_link($device, $text = null, $vars = array(), $start = 0
 
     $class = devclass($device);
     if (!$text) {
-        $text = $device['hostname'];
+        $text = format_hostname($device['hostname']);
```

Comment:
Did i miss something?
As i see $text will be formated directly after this if statement with
`$text = format_hostname($device, $text);`

Reply:
Fixed